test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable protractor/no-describe-selectors */
+import { browser, by } from 'protractor';
+import { config } from '../protractor/local.config';
```

Comment:
You are getting always local file, what happend when it's running in headless mode?